### PR TITLE
Update xrp.toml

### PR DESCRIPTION
**Pull request by biasgooseRX**

### CODE REVIEW

File: data/ecosystems/x/xrp.toml [hunk 1]
```diff
@@ -1679,3 +1679,1701 @@ url = "https://github.com/zshahzadpumacy/Redimi-Giftcards-SmartContract"
...
+url = "https://github.com/selcukemiravci/CharityDAO"
+
+[[repo]]
+url = "https://github.com/SecuX/secux-js"
+
+[[repo]]
+url = "https://github.com/rossgeesman/cryptodex"
+
+[[repo]]
+url = "https://github.com/futureversecom/trn-examples"
```
Comment (eherrerosj) [inline data/ecosystems/x/xrp.toml:1894]: I've been seeing some repos which I'm not sure are building on top of Ripple. Take this as an example, could you explain the rationale or algorithm you are using to find these repos? Thinking how to filter them a bit. I have one idea: feel free to ping me on telegram (same handle as my gh user)

**Review by eherrerosj — COMMENTED** [review]
left a comment

File: data/ecosystems/x/xrp.toml (hunk 1 above)
Comment (biasgooseRX) [reply data/ecosystems/x/xrp.toml:1894]: hey @eherrerosj! Thanks for flagging that. These are part of the larger XRP Ledger ecosystem. The futureverse (Root Network) is a type of sidechain to the XRP Ledger and thus I included it. I will ping you on telegram if you would like to discuss it further, thanks!

**Comment by biasgooseRX** [comment]
@eherrerosj I have updated this to remove root network repos and will in a later date add them as a subecosystem

**Comment by eherrerosj** [comment]
> @eherrerosj I have updated this to remove root network repos and will in a later date add them as a subecosystem

Cool thx. Validating the rest now and hoping to get back to you in the next couple days

**Review by eherrerosj — APPROVED** [review]
awesome work! thanks for the contribution

**Pull request merged**